articles/active-directory-b2c/active-directory-b2c-devquickstarts-graph-dotnet.md

```diff
@@ -56,6 +56,11 @@ Now you need to configure your application to get all the required permissions t
 
 You now have an application that has permission to create, read and update users from your B2C tenant.
 
+> [!NOTE]
+> This can take some time to process.  NEED_RECOMMENDED_TIME_WAIT
```

Comment:
Please update the note to: "Granting permissions make take a few minutes to fully process."

Reply:
@spottedmahn Thanks for the contribution. Could you edit file in your branch to the suggested text from Parakh? We don't have permission to your fork. Once it's updated @parakhj can request to merge it.

Reply:
@PatAltimore done 😄